client/src/test/RestaurantCard.test.jsx

```diff
@@ -0,0 +1,15 @@
+import { configure, shallow } from 'enzyme';
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16/build';
+import { mockRestaurants } from '../consts';
+import { RestaurantCard } from '../RestaurantList/RestaurantCard';
+
+configure({ adapter: new Adapter() });
+
+describe('RestaurantCard', () => {
+  it('should render correctly', () => {
+    let mockRestaurant = mockRestaurants[0];
+    let restaurantCard = shallow(<RestaurantCard restaurant={mockRestaurant} />);
+    expect(restaurantCard).toMatchSnapshot();
```

Comment:
Check your snapshots, they look like a mess.
Consult [enzyme-to-json](https://www.npmjs.com/package/enzyme-to-json) for a better readability

Reply:
https://www.youtube.com/watch?v=HBxt_v0WF6Y